README.md

```diff
@@ -33,3 +33,83 @@ Your pull requests are welcome but please keep them manageable and focussed.
 Equally your input on any pull requests we have in flight at any given time is invaluable to us, so please get involved.
 
 Thanks! :grin:
+
+
+## Running the example
+
+* Clone the repo
+* cd to `example` folder
+* run `flutter packages get` to install dependencies
+* `flutter run` will start the application on connected android / iOS device
+
+
+## Using the Realtime API
+
+##### Import the package
+```dart
+import 'package:ably_flutter_plugin/ably.dart' as ably;
+```
+
+##### Create an Ably instance
+```dart
+final ably.Ably ablyPlugin = ably.Ably();
+```
+
+##### create a ClientOptions
+```dart
+final clientOptions = ably.ClientOptions.fromKey("<YOUR APP KEY>");
+clientOptions.environment = 'sandbox';
```

Comment:
Please remove `sandbox`.  This is only intended for client SDK developers, no customer would use `sandbox` as they cannot create API keys in sandbox

Reply:
Done@https://github.com/ably/ably-flutter/commit/d46b88ed11b34b958d0a3483d9001773ef2fb7dd

Reply:
I would go further and remove the `environment` line entirely, certainly for high-level documentation like this which is aimed at users (i.e. Flutter app developers).

Reply:
Removed `environment` from README c777214e920237f450d5e27b80503becd6bb641e